nebula-algorithm/README-CN.md

```diff
@@ -0,0 +1,75 @@
+# 欢迎使用 nebula-algorithm
```

Comment:
Nebula Algorithm. Please modify the product names in the following content as well.

Reply:
sure.